packages/synthchron-tauri/src/components/react-flow/DragAndDropWrapper.tsx

```diff
@@ -51,7 +51,10 @@ export const DragAndDropWrapper = () => {
       id: '', // Will be overwritten by addNode
       type: type,
       position,
-      data: { label: `${type == 'place' ? 'p' : 't'}`, store: 0 },
+      data: {
+        label: `${type == 'place' ? 'place' : 'transition'}`,
+        storage: 0,
```

Comment:
Is renaming "store" to "storage" a breaking change? Is "store" being used anywhere else in the code? If so, please also update all other occurences